tests/test_build.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+
+from sqlglot import parse_one
+from sqlglot import expressions as exp
+from sqlglot.expressions import Select
+
+
+class TestBuild(unittest.TestCase):
+    def test_create(self):
+        table = exp.Table(this="tbl", db="db")
```

Comment:
We'd probably want a cleaner interface here. `this=...` is a bit clunky.

Reply:
i don't like the column part... i'd rather do something like

table = exp.Table

Select.from(table).where("x = 1")